01_Data/src/layers/sequelize/repository/Tariff.ts

```diff
@@ -23,14 +23,20 @@ export class SequelizeTariffRepository extends SequelizeRepository<Tariff> imple
   }
 
   async createOrUpdateTariff(tariff: Tariff): Promise<Tariff> {
-    const [storedTariff, _tariffCreated] = await this.upsert(
-      Tariff.build({
-        stationId: tariff.stationId,
-        unit: tariff.unit,
-        price: tariff.price,
-      }),
-    );
-    return storedTariff;
+    // TODO check if searchg correclty
```

Comment:
Yes, it would work for the current tariff workaround. I found the unique constraint is missing in the Tariff model. Could you please help add it in 01_Data/src/layers/sequelize/model/Tariff/Tariffs.ts? It looks like,
```
  @Index
  @Column({
    type: DataType.STRING,
    unique: 'stationId_unit'
  })
  declare stationId: string;

  @Column({
    type: DataType.STRING,
    unique: 'stationId_unit'
  })
  declare unit: TariffUnitEnumType;
```

Reply:
Sure will add the constraints to the model :)